addons/stock/wizard/stock_replenishment_info.py

```diff
@@ -46,35 +52,42 @@ def _compute_json_lead_days(self):
 
     @api.depends('orderpoint_id')
     def _compute_json_replenishment_history(self):
+        settings = self.env["res.config.settings"].sudo().search([])
```

Comment:
@victorvermot as this is a TransientModel you should not search for records of this model. They exist only in memory.
However, the value of this setting is stored in the company record